packages/eslint-plugin-pf-codemods/lib/helpers.js

```diff
@@ -493,15 +501,33 @@ function addCallbackParam(componentsArray, propMap) {
                 }
                 const { type, params } = propProperties;
 
+                const parameterConfig = propMap[attribute.name.name];
+                const isParamAdditionOnly = typeof parameterConfig === "string";
+                const newOrDefaultParamName = isParamAdditionOnly ? parameterConfig : parameterConfig.defaultParamName
+
+                let potentialParamMatchers = `(_?${stripLeadingUnderscore(newOrDefaultParamName)})`
+                const otherMatchersString = parameterConfig.otherMatchers?.toString().slice(1, -1)
```

Comment:
```suggestion
                const otherMatchersString = parameterConfig.otherMatchers?.toString().slice(1, -1);
```

Reply:
Prettier did a lot of things that I don't love formatting wise, but your semicolons are in now 🤣